### PR TITLE
Feanil/ecomstack changes

### PR DESCRIPTION
**Pull request by feanil**
Changes to make things work with the ecomstack.

### CODE REVIEW

File: requirements/not_editable.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Oscar requirements
+git+https://github.com/edx/django-oscar.git@1ce871a29b97789354b422ca559de956c6762aee#egg=django-oscar
+git+https://github.com/edx/django-oscar-extensions.git@93d3286ecf68039fc360aa92fddd81bf456f532e#egg=django-oscar-extensions
```
Comment (rlucioni) [inline requirements/not_editable.txt]: Latest hash of extensions is `1bd87db2b0703f55c56d1266b09da59aa9d8439e`.

**Comment by rlucioni** [comment]
One minor comment, otherwise :+1: assuming the tests pass.

File: requirements/editable.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Local editable sources of django-oscar and django-oscar-extensions
+-e ../../depends/django-oscar
```
Comment (clintonb) [inline requirements/editable.txt:2]: The README should be updated informing developers of the need to have these repos checked out in a `depends` directory.

**Comment by clintonb** [comment]
:+1: after note addressed.

**Comment by feanil** [comment]
Readme updated.

**Pull request merged**